Dockerfile

```diff
@@ -11,15 +11,15 @@ RUN apk --no-cache add \
 	make
 
 ENV COREDNS_VERSION v1.6.9
-ENV DNSSEEDER_VERSION v0.2.1
+ENV DNSSEEDER_VERSION v0.2.2
```

Comment:
Check if this is the [latest dnsseeder](https://github.com/ZcashFoundation/dnsseeder) version

Reply:
✅